pkgs/development/haskell-modules/configuration-ghc-8.4.x.nix

```diff
@@ -0,0 +1,761 @@
+{ pkgs, haskellLib }:
+
+with haskellLib;
+
+self: super: {
+
+  # Use the latest LLVM.
+  inherit (pkgs) llvmPackages;
+
+  # Disable GHC 7.11.x core libraries.
```

Comment:
@peti, pushed an attempt at this update.  One question is -- what is `bin-package-db` -- did previous GHC's indeed used to have this package or not?  I retained its entry, because it sounded magical enough..